packages/core/src/application.ts

```diff
@@ -47,24 +47,19 @@ export class Application extends Context {
       }
     }
 
-    if (appConfig && appConfig.sequences) {
-      for (const sequence of appConfig.sequences) {
-        // TODO(superkhau): Need to figure a way around this hack,
-        //   `componentClassName.constructor.name` + `componentClassName.name`
-        //   doesn't work
-        const sequenceClassName = sequence.toString().split(' ')[1];
-        this.bind(`sequence.${sequenceClassName}`).toClass(sequence);
-      }
-    }
+    this._bindSequence(options);
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) =>
       this._handleHttpRequest(req, res);
 
     this.bind('logError').to(this._logError.bind(this));
+  }
 
+  protected _bindSequence(options?: ApplicationOptions): void {
```

Comment:
The options is already available as `this.options`. Not need to pass in as argument.